root_github.tf

```diff
@@ -85,3 +85,17 @@ module "github_checksum_repository" {
     WORKFLOW_PAT       = data.aws_ssm_parameter.workflow_pat.value
   }
 }
+
+module "github_generated_grapqhl_environment" {
```

Comment:
Spelling of module name, should be: `..._graphql_...`

Reply:
Oops, I've changed it.